Gemfile

```diff
@@ -11,6 +11,6 @@ group :test do
 end
 
 group :system_tests do
-  gem 'beaker', '~> 2.14'
+  gem 'beaker', '~> 2.15'
```

Comment:
you mention it's broken for beaker 2.16, should we be pinning to that instead of 2.15 as the minimum version?